Binary files com.discord-1144-base/build/apk/classes2.dex and com.discord-1144/build/apk/classes2.dex differ
Binary files com.discord-1144-base/build/apk/res/layout/flex_input_widget.xml and com.discord-1144/build/apk/res/layout/flex_input_widget.xml differ
Binary files com.discord-1144-base/dist/com.discord-1144.apk and com.discord-1144/dist/com.discord-1144.apk differ
diff -crB com.discord-1144-base/res/layout/flex_input_widget.xml com.discord-1144/res/layout/flex_input_widget.xml
*** com.discord-1144-base/res/layout/flex_input_widget.xml	2020-05-17 16:38:23.695244181 +0300
--- com.discord-1144/res/layout/flex_input_widget.xml	2020-05-17 18:35:31.125235116 +0300
***************
*** 8,14 ****
      <RelativeLayout android:animateLayoutChanges="true" style="@style/UiKit.ViewGroup">
          <LinearLayout android:id="@id/flex_input_left_btns_container" android:layout_width="wrap_content" android:layout_marginLeft="5.0dip" android:layout_marginTop="@dimen/uikit_spacing_medium" android:layout_marginBottom="@dimen/uikit_spacing_medium" android:layout_alignParentLeft="true" android:layout_centerVertical="true" style="@style/UiKit.ViewGroup.LinearLayout.Horizontal">
              <androidx.appcompat.widget.AppCompatImageButton android:layout_gravity="center_vertical" android:id="@id/flex_input_expand_btn" android:padding="@dimen/uikit_spacing_medium" android:visibility="gone" android:layout_width="wrap_content" android:layout_height="wrap_content" android:src="@drawable/ic_carot_right_24dp" android:tint="?colorInteractiveNormal" android:contentDescription="@string/expand_buttons" style="@style/FlexInput.Input" />
!             <androidx.appcompat.widget.AppCompatImageButton android:id="@id/flex_input_camera_btn" android:background="@drawable/drawable_circle_black" android:padding="@dimen/uikit_spacing_medium" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="6.0dip" android:layout_marginRight="@dimen/uikit_spacing_medium" android:src="@drawable/ic_camera_24dp" android:tint="?colorInteractiveNormal" android:contentDescription="@string/camera" android:backgroundTint="?colorBackgroundSecondaryAlt" style="@style/FlexInput.Input" />
              <androidx.appcompat.widget.AppCompatImageButton android:id="@id/flex_input_gallery_btn" android:background="@drawable/drawable_circle_black" android:padding="@dimen/uikit_spacing_medium" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginRight="@dimen/uikit_spacing_medium" android:src="@drawable/ic_gallery_24dp" android:tint="?colorInteractiveNormal" android:contentDescription="@string/attachment_photos" android:backgroundTint="?colorBackgroundSecondaryAlt" style="@style/FlexInput.Input" />
          </LinearLayout>
          <LinearLayout android:id="@id/flex_input_main_input_container" android:background="@drawable/drawable_bg_corners_20dp" android:paddingLeft="12.0dip" android:paddingRight="@dimen/uikit_spacing_small" android:layout_marginTop="@dimen/uikit_spacing_medium" android:layout_marginBottom="@dimen/uikit_spacing_medium" android:layout_toLeftOf="@id/flex_input_send_btn_container" android:layout_toRightOf="@id/flex_input_left_btns_container" android:layout_centerVertical="true" android:animateLayoutChanges="true" android:backgroundTint="?colorBackgroundSecondaryAlt" style="@style/UiKit.ViewGroup.LinearLayout.Horizontal">
--- 8,14 ----
      <RelativeLayout android:animateLayoutChanges="true" style="@style/UiKit.ViewGroup">
          <LinearLayout android:id="@id/flex_input_left_btns_container" android:layout_width="wrap_content" android:layout_marginLeft="5.0dip" android:layout_marginTop="@dimen/uikit_spacing_medium" android:layout_marginBottom="@dimen/uikit_spacing_medium" android:layout_alignParentLeft="true" android:layout_centerVertical="true" style="@style/UiKit.ViewGroup.LinearLayout.Horizontal">
              <androidx.appcompat.widget.AppCompatImageButton android:layout_gravity="center_vertical" android:id="@id/flex_input_expand_btn" android:padding="@dimen/uikit_spacing_medium" android:visibility="gone" android:layout_width="wrap_content" android:layout_height="wrap_content" android:src="@drawable/ic_carot_right_24dp" android:tint="?colorInteractiveNormal" android:contentDescription="@string/expand_buttons" style="@style/FlexInput.Input" />
!             <androidx.appcompat.widget.AppCompatImageButton android:id="@id/flex_input_camera_btn" android:visibility="gone" android:background="@drawable/drawable_circle_black" android:padding="@dimen/uikit_spacing_medium" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="6.0dip" android:layout_marginRight="@dimen/uikit_spacing_medium" android:src="@drawable/ic_camera_24dp" android:tint="?colorInteractiveNormal" android:contentDescription="@string/camera" android:backgroundTint="?colorBackgroundSecondaryAlt" style="@style/FlexInput.Input" />
              <androidx.appcompat.widget.AppCompatImageButton android:id="@id/flex_input_gallery_btn" android:background="@drawable/drawable_circle_black" android:padding="@dimen/uikit_spacing_medium" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginRight="@dimen/uikit_spacing_medium" android:src="@drawable/ic_gallery_24dp" android:tint="?colorInteractiveNormal" android:contentDescription="@string/attachment_photos" android:backgroundTint="?colorBackgroundSecondaryAlt" style="@style/FlexInput.Input" />
          </LinearLayout>
          <LinearLayout android:id="@id/flex_input_main_input_container" android:background="@drawable/drawable_bg_corners_20dp" android:paddingLeft="12.0dip" android:paddingRight="@dimen/uikit_spacing_small" android:layout_marginTop="@dimen/uikit_spacing_medium" android:layout_marginBottom="@dimen/uikit_spacing_medium" android:layout_toLeftOf="@id/flex_input_send_btn_container" android:layout_toRightOf="@id/flex_input_left_btns_container" android:layout_centerVertical="true" android:animateLayoutChanges="true" android:backgroundTint="?colorBackgroundSecondaryAlt" style="@style/UiKit.ViewGroup.LinearLayout.Horizontal">
***************
*** 21,24 ****
          <TextView android:gravity="center" android:id="@id/flex_input_cannot_send_text" android:background="?colorBackgroundSecondary" android:paddingLeft="12.0dip" android:paddingTop="@dimen/uikit_spacing_xlarge" android:paddingRight="12.0dip" android:paddingBottom="@dimen/uikit_spacing_xlarge" android:focusable="true" android:visibility="gone" android:clickable="true" android:layout_height="wrap_content" android:text="@string/no_send_messages_permission_placeholder" android:layout_alignParentStart="true" android:layout_alignParentEnd="true" style="@style/UiKit.TextView" />
      </RelativeLayout>
      <FrameLayout android:orientation="vertical" android:id="@id/flex_input_emoji_container" android:visibility="gone" android:layout_width="fill_parent" android:layout_height="@dimen/default_keyboard_height" />
! </LinearLayout>
\ No newline at end of file
--- 21,24 ----
          <TextView android:gravity="center" android:id="@id/flex_input_cannot_send_text" android:background="?colorBackgroundSecondary" android:paddingLeft="12.0dip" android:paddingTop="@dimen/uikit_spacing_xlarge" android:paddingRight="12.0dip" android:paddingBottom="@dimen/uikit_spacing_xlarge" android:focusable="true" android:visibility="gone" android:clickable="true" android:layout_height="wrap_content" android:text="@string/no_send_messages_permission_placeholder" android:layout_alignParentStart="true" android:layout_alignParentEnd="true" style="@style/UiKit.TextView" />
      </RelativeLayout>
      <FrameLayout android:orientation="vertical" android:id="@id/flex_input_emoji_container" android:visibility="gone" android:layout_width="fill_parent" android:layout_height="@dimen/default_keyboard_height" />
! </LinearLayout>
diff -crB com.discord-1144-base/smali_classes2/com/lytefast/flexinput/fragment/FlexInputFragment.smali com.discord-1144/smali_classes2/com/lytefast/flexinput/fragment/FlexInputFragment.smali
*** com.discord-1144-base/smali_classes2/com/lytefast/flexinput/fragment/FlexInputFragment.smali	2020-05-17 16:38:28.831942620 +0300
--- com.discord-1144/smali_classes2/com/lytefast/flexinput/fragment/FlexInputFragment.smali	2020-05-17 18:39:05.170080075 +0300
***************
*** 793,799 ****
  
      if-nez v4, :cond_4
  
!     const/4 v10, 0x0
  
      goto :goto_1
  
--- 793,799 ----
  
      if-nez v4, :cond_4
  
!     const/16 v10, 0x8
  
      goto :goto_1
  
***************
*** 810,816 ****
  
      if-eqz v4, :cond_5
  
!     const/4 v10, 0x0
  
      goto :goto_2
  
--- 810,816 ----
  
      if-eqz v4, :cond_5
  
!     const/16 v10, 0x8
  
      goto :goto_2
  
***************
*** 832,838 ****
      goto :goto_3
  
      :cond_6
!     const/16 v4, 0x8
  
      :goto_3
      invoke-virtual {v5, v4}, Landroid/view/View;->setVisibility(I)V
--- 832,838 ----
      goto :goto_3
  
      :cond_6
!     const/16 v4, 0x0
  
      :goto_3
      invoke-virtual {v5, v4}, Landroid/view/View;->setVisibility(I)V

